thirdeye/thirdeye-frontend/app/styles/shared/_styles.scss

```diff
@@ -529,6 +529,16 @@ body {
     font-weight: 500;
     color: app-shade(black, 8);
     position: relative;
+
+    &--limited {
+      max-width: 155px;
+    }
+
+    &--last {
```

Comment:
can't this be handled with `:last-child` ? 
https://developer.mozilla.org/en-US/docs/Web/CSS/:last-child

Reply:
yes. replaced.